Gruntfile.js

```diff
@@ -535,13 +535,11 @@ module.exports = function(grunt) {
       },
       'shared-lib-unit': {
         cmd: () => {
-          return getSharedLibDirs()
-            .map(
-              lib =>
-                `echo Testing shared library: ${lib} &&
-                 (cd shared-libs/${lib} && npm ci && npm test)`
-            )
-            .join(' && ');
+          const sharedLibs = getSharedLibDirs();
```

Comment:
The `dedupe` actually deletes `node_modules` from the `shared-libs` folders, and because some modules require  others, testing fails. So we `ci` all of them first.